Tests/NimbleTests/objc/ObjCEndWithTest.m

```diff
@@ -13,7 +13,6 @@ - (void)testPositiveMatches {
     expect(@"hello world!").toNot(endWith(@"hello"));
     expect(array).to(endWith(@2));
     expect(array).toNot(endWith(@1));
-    expect(@1).toNot(contain(@"foo"));
```

Comment:
Testing `contain` matcher in `ObjCEndWithTest.m` is odd (and the fact the test is passing is odd as well) 😛